app/models/region.rb

```diff
@@ -19,8 +19,8 @@ class Region < ApplicationRecord
   # A label is a sequence of alphanumeric characters and underscores.
   # (In C locale the characters A-Za-z0-9_ are allowed).
   # Labels must be less than 256 bytes long.
-  def name_to_path_label
-    name.gsub(/\W/, "_").slice(0, MAX_LABEL_LENGTH)
+  def path_label
+    slug.gsub(/\W/, "_").slice(0, MAX_LABEL_LENGTH)
```

Comment:
I suppose we still need to do this after Friendly ID, even though it prefers underscores because of [uniqueness](https://github.com/norman/friendly_id/blob/master/lib/friendly_id/slugged.rb#L100) and probably just general care-taking of the health of this path.